src/frontend/types/VideoPlayerInterface.ts

```diff
@@ -0,0 +1,6 @@
+export type VideoPlayerType = 'plyr';
```

Comment:
Maybe use an `enum`?

Reply:
I don't really to use it like an enum. For now I should prefer not usin an enum. Maybe later it will be better but for now I don't think so.